src/ducks/apps/components/ApplicationRouting/index.jsx

```diff
@@ -40,9 +39,13 @@ export class ApplicationRouting extends Component {
   }
 
   render() {
-    const { isFetching, parent } = this.props
+    const { isFetching, parent, intentData, onTerminate } = this.props
     return (
-      <div className="sto-modal-page" ref={this.mainPage}>
+      <div
+        className="sto-modal-page"
+        {...(!!intentData && { style: { maxHeight: '100%' } })}
```

Comment:
we should avoid to create inline object for style inside the render. Because each time React call the render, each time we'll create a new object.

Reply:
You still do it. You define the var inside the render so this is the same thing as before.

Reply:
Corrigé 👍